argocd/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine as build
+# renovate: datasource=github-releases depName=argoproj/argo-cd
+ENV VERSION=2.10.2
+ENV CHECKSUM=a8da9fa1ea7b7072007f535d39526ad0c4a8b8eb58779b48774126335c576187
```

Comment:
this is interesting, Im not sure if we can make renovate download the new checksum as well as part of the autobump. In any case this is good, we will see if we can make this work when we reach the problem :D

Reply:
I set checksum var into RUN steps, with testing TARGETARCH. it could be a viable solution ?

Reply:
I dont know TBH, but we should not worried about this too much rigth now :D

Reply:
Agreed. Better over perfect - we might redo some of this in the future anyway by building versioned containers instead of what is essentially just `latest`.